yellowbrick/features/__init__.py

```diff
@@ -26,3 +26,4 @@
 from .importances import FeatureImportances, feature_importances
 from .rfecv import RFECV, rfecv
 from .manifold import Manifold, manifold_embedding
+from .base import TargetType
```

Comment:
Thought it would be important to have this at the top level, I also imported it in `yellowbrick/target/__init__.py` so it's available from both packages.